app/decorators/controllers/courses_controller_decorator.rb

```diff
@@ -127,13 +129,16 @@ def strongmind_settings
   def strongmind_update
     instructure_update
     return if params[:course].blank?
+    session[:relock_warning] = false
+    passing_thresholds_edited = params.select{|k,v| k.match(/(_passing_threshold_edited)/) && v == "true"}.present?
```

Comment:
I know I wrote this, but looking back, we _may_ be able to make the `passing_thresholds_edited` into a controller method that is called by `strongmind_update` just to clean up this method a little bit more. 

Not a blocking change but may be worth trying...